### ISSUE TITLE
multi same task

### ISSUE THREAD

**Issue by yzbx** [ISCAS007/torchseg#22]
# mode
suppose depth = 3
## 333, 111
```
https://g.gravizo.com/svg?
digraph G {
feature -> mid_conv1_1 -> mid_conv1_2 ->mid_conv1_3 -> out1
feature -> mid_conv2_1 -> mid_conv2_2 ->mid_conv2_3 -> out2
feature -> mid_conv3_1 -> mid_conv3_2 ->mid_conv3_3 -> out3
}
```

![333](https://g.gravizo.com/svg?%20digraph%20G%20{%20feature%20-%3E%20mid_conv1_1%20-%3E%20mid_conv1_2%20-%3Emid_conv1_3%20-%3E%20out1%20feature%20-%3E%20mid_conv2_1%20-%3E%20mid_conv2_2%20-%3Emid_conv2_3%20-%3E%20out2%20feature%20-%3E%20mid_conv3_1%20-%3E%20mid_conv3_2%20-%3Emid_conv3_3%20-%3E%20out3%20})

**Comment by yzbx** [comment]
# experiment results
## cifar10
![screenshot from 2018-09-25 13-28-51](https://user-images.githubusercontent.com/5005182/45995465-be187200-c0ca-11e8-959a-d7160b670288.png)

| mode | dataset | epoch | max train acc | max val acc | conclusion |
| - | - | - | - | - | - |
321 | cifar10 | 10 | ? | ?

**Comment by yzbx** [comment]
# [use state-of-the-art model](https://github.com/aaron-xichen/pytorch-playground)
## pytorch-playerground
### cifar10 (default setting)
`python cifar/train.py --gpu=2`
>Test set: Average loss: 0.3930, Accuracy: 9337/10000 (93%)

### cifar100
`python cifar/train.py --gpu=1 --type=cifar100 --batch_size=50`
>Test set: Average loss: 1.9802, Accuracy: 7365/10000 (73%)
